api/src/routes.php

```diff
@@ -70,7 +70,7 @@
         Flight::route('PUT|PATCH /localisation', [LocalisationController::class, 'update']);
 
         Flight::route('GET /', [ProfileController::class, 'index']);
-        Flight::route('PUT|PATCH /', [ProfileController::class, 'update']);
+        Flight::route('PUT|POST /', [ProfileController::class, 'update']);
```

Comment:
```suggestion
        Flight::route('PUT|PATCH/', [ProfileController::class, 'update']);
```
Create new route for photos